pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: observable
-version: 0.20.0
+version: 0.21.0
```

Comment:
should be 0.20.1 for a new functionality addition to a 0.x.y+z versioned pacakge.